cmd/pilot-agent/main.go

```diff
@@ -119,6 +125,30 @@ var (
 			proxyConfig.StatsdUdpAddress = statsdUDPAddress
 			proxyConfig.ProxyAdminPort = int32(proxyAdminPort)
 
+			var pilotSAN []string
```

Comment:
put this in a function to make `main.go` more readable

Reply:
this pr got merged, will fix in another PR